packages/scraper-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@apify/scraper-tools",
-    "version": "1.0.0",
+    "version": "1.0.1",
```

Comment:
why? this will release canary 1.0.2, are you aware of that?

Reply:
But I just told you on Slack that once I bumped the version to 1.0.1 I was able to install the stuff locally - I thought you kinda confirmed that if it helps - to do so. Ok, reverting 👍

(although now events test does not pass for no reason).